app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="recipedetails">
+  <h1><%= @recipe.name %></h1>
+
+  <% if current_user == @recipe.user || @recipe.public %>
+    <%= render 'recipe_details', recipe: @recipe %>
+  <% else %>
+    <p>This recipe is private.</p>
+  <% end %>
+
+  <div class="content-container">
+    <div class="left-section">
+      <div class="names">
+        <p class="time"><strong><%= @recipe.editing_times? ? @recipe.preparation_time : "preparation_time: #{@recipe.preparation_time} hours" %></p>
+        <p class="time"><%= @recipe.editing_times? ? @recipe.cooking_time : "cooking_time: #{@recipe.cooking_time} hours" %></p>
+        <p class="steps">Steps go here...</p>
+      </div>
+    </div>
+
+    <div class="right-section">
+      <div class="buttons" id="is_public">
+        <%= button_to(@recipe.public? ? "Make it Private" : "Make it Public", toggle_public_recipe_path(@recipe), method: :patch, class: "toggle-button #{'private' unless @recipe.public?}") %>
+      </div>
+    </div>
+  </div>
+  <div class="buttons">
+  <%= link_to "Generate shopping list", general_shopping_list_path, class: 'btn btn-action' %>
+	<% if current_user == @recipe.user %>
+	<%= link_to "Add ingredient", new_recipe_recipe_food_path(recipe_id: @recipe.id), class: 'btn1 btn-action' %>
+	<% end %>
+  </div>
```

Comment:
I don't have to build the all project because it is a group project and I'm working alone.
Adding new ingredients and adding new food are some features I will have to work on.
The most important for me is to complete a team task but not the all project.

Reply:
- [ ] I understand that you are working alone on this project. It isn't an easy feat to accomplish and I give you credit for this. 👏. However, when working on a group capstone independently, you would have to implement all the requirements designated for two students. This means that you would have to implement all the requirements that you and your former partner would originally execute. In view of this, I would advise that you do the following:

- Create the `Foods#Index` page
- Add a `Add Food` button on the `Foods#Index` page that you will create and link it to the `Foods#New` page that you already created
- Linking the food items already created to the form page for creating an ingredient on the `Food_Recipes#New` page. 

#### N.B: Kindly implement the change suggested by @abdulhamiid 👍 😎